client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -216,6 +227,17 @@ module.exports = React.createClass( {
 					</FormButton>
 
 					<FormButton
+						className="security-2fa-backup-codes-list__save"
+						isPrimary={ false }
+						onClick={ function( event ) {
+							analytics.ga.recordEvent( 'Me', 'Clicked On 2fa Save Backup Codes Button' );
+							this.saveCodesToFile( event );
+						}.bind( this ) }
```

Comment:
to avoid the bind and inline function here, how about moving the analytics call to `saveCodesToFile`?